SoftLayer/CLI/subnet/detail.py

```diff
@@ -56,7 +57,9 @@ def cli(env, identifier, no_vs, no_hardware):
 
     ip_table = formatting.KeyValueTable(['id', 'ip', 'note'])
     for address in ip_address:
-        ip_table.add_row([address.get('id'), address.get('ipAddress'), address.get('note')])
+        ip_table.add_row([address.get('id'),
+                          address.get('ipAddress') + '/' +
+                          subnet['networkVlan']['primaryRouter']['fullyQualifiedDomainName'], address.get('note')])
```

Comment:
I want the `hardware` or `virtualGuest` fullyQualifiedDomainName added here, not the primaryRouter name